### PR TITLE
initial toc

### PR DESCRIPTION
**Pull request by chicco785**

### CODE REVIEW

**Review by taliaga — DISMISSED** [review]

**Review by taliaga — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,3 +23,9 @@ a pull request to the repository, you will be able to observe the results of
 the compliancy verification in your PR. Merge will be only possible if CI process
 is passed successfully.
 
+Should you have a solution your self, feel free to make a pull request!
```
Comment (taliaga) [inline README.md]: typo: yourself

File: docs/future.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Planned Future Evolutions
+
+This section describe the planned evolution for the Lab (and hence for the documentation).
```
Comment (taliaga) [inline docs/future.md]: typo: describes

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Welcome to the Guidelines to Join FIWARE Lab
+
+This documentation describes the process to set-up a FIWARE Lab Node
+and to connect it to the FIWARE Lab.
```
Comment (taliaga) [inline docs/index.md]: "...and connect it..." would read better

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Table of Contents
+
+1. [Introduction](introduction.md)
+1. [Setting-up a node](node-set-up)
+    1. [Requirements](node-set-up/requirements.md)
+    1. [Installation](node-set-up/installation.md)
+1. [Connecting a node](node-connection)
+    1. [Connecting your Node to FIWARE Lab](node-connection/fiware-lab.md)
```
Comment (taliaga) [inline docs/index.md]: consistency: node or Node?

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    1. [Registering your Node in FI-Health](node-connection/fi-health.md)
+    1. [Registering your Node in the Infographic](node-connection/infographic.md)
+1. [Disconnecting a node](node-disconnection)
+    1. [De-Registering your Node to FIWARE Lab](node-disconnection/fiware-lab.md)
```
Comment (taliaga) [inline docs/index.md]: De-Registering -> Deregistering, and I'd say preposition here is "from", not "in" neither "to".

File: docs/node-connection/fiware-lab.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Connecting your Node to FIWARE Lab
+
+This section describe how to connect a node to FIWARE Lab
```
Comment (taliaga) [inline docs/node-connection/fiware-lab.md]: typo: describes

File: docs/node-disconnection/fiware-lab.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# De-Registering your Node to FIWARE Lab
+
+This section describe how to de-register your Node to FIWARE Lab
```
Comment (taliaga) [inline docs/node-disconnection/fiware-lab.md]: ditto

File: docs/index.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+## Table of Contents
+
+1. [Introduction](introduction.md)
+1. [Setting-up a node](node-set-up)
+    1. [Requirements](node-set-up/requirements.md)
+    1. [Installation](node-set-up/installation.md)
+1. [Connecting a node](node-connection)
+    1. [Connecting your Node to FIWARE Lab](node-connection/fiware-lab.md)
+    1. [Registering your Node in FI-Health](node-connection/fi-health.md)
```
Comment (taliaga) [inline docs/index.md]: consistency: Connecting vs Registering? If there's a difference, why there's no Disconnecting below?

**Review by taliaga — CHANGES_REQUESTED** [review]

File: README.md [hunk 9]
```diff
@@ -1,8 +1,19 @@
-# join-fiware-lab-guide
+# How to Join FIWARE Lab
+
+[![Documentation Status](https://readthedocs.org/projects/join-fiware-lab-guide/badge/?version=latest)](http://join-fiware-lab-guide.readthedocs.io/en/latest/?badge=latest)
 
 In this repository you can find the up-to-date guide on how your own OpenStack
-data center can Join FIWARE Lab, a community cloud offering free resources to
-FIWARE developers.
+data center can Join [FIWARE Lab](http://lab.fiware.org), a community cloud
```
Comment (flopezag) [inline README.md]: mmm, this link goes at the end to the account.lab.fiware.org and not to the cloud.lab.fiware.org. What is the purpose of this link?

Comment (flopezag) [reply README.md]: Probably it is something that has to be changed outside of the repo, let me know.

File: README.md [hunk 10]
```diff
@@ -1,8 +1,19 @@
...
+offering free resources to [FIWARE](https://www.fiware.org) developers.
+
+Documentation is tagged according to FIWARE Lab release plan.
+
+## Contributions
```
Comment (flopezag) [inline README.md]: Wow, apparently it is correct but it is not shown correctly in the GitHub, take care about it

File: docs/index.md (hunk 8 above)
Comment (flopezag) [reply docs/index.md]: Connecting are the different steps to connect the services to the central keystone. Registering is more related to the operations to join the Sanity Checks and Monitoring tools of the FIWARE Lab.

File: docs/index.md (hunk 3 above)
Comment (flopezag) [reply docs/index.md]: 👍

**Review by flopezag — COMMENTED** [review]
General comment, We plan to put inside this repository the corresponding scripts to help in the different steps, therefore some description about those scripts should be provided in the corresponding .md file.

File: README.md (hunk 9 above)
Comment (chicco785) [reply README.md]: changed to http://help.lab.fiware.org

**Pull request merged**